README.md

```diff
@@ -375,6 +375,8 @@ the following events:
 * `"tutorialVisibilityChange"` - triggered when the tutorial preview is turned on or off. It includes an `Object` with a `visibility` property that indicates whether the tutorial is visible.
 * `"inspectorChange"` - triggered whenever the inspector changes from enabled to disabled, or vice versa. It includes an `Object` with an `enabled` property set to `true` or `false`.
 * `"autoUpdateChange"` - triggered whenever the auto update preference changes from enabled to disabled, or vice versa. It includes an `Object` with a `autoUpdate` property set to `true` or `false`
+* `"projectDirty"` - triggered whenever the current file has been changed. It includes an `Object` with the path to the current file.
+* `"projectSaved"` - triggered whenever the changes are saved to the local computer are completed.
```

Comment:
Similarly, this should say that we trigger it when all files in the project have been saved, but don't say "to the local computer" because that's not really true.  It's saved to the filesystem in the browser.